homeassistant/components/shelly/device_trigger.py

```diff
@@ -108,9 +108,9 @@ async def async_validate_trigger_config(
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, Any]]:
+) -> list[dict[str, str]]:
     """List device triggers for Shelly devices."""
-    triggers: list[dict[str, Any]] = []
+    triggers: list[dict[str, str]] = []
```

Comment:
The type of the `triggers` parameter in `append_input_triggers` doesn't match now.

Reply:
https://github.com/home-assistant/core/pull/72146